test/avocado/run-tests

```diff
@@ -305,14 +305,16 @@ class SeleniumTestSuite(AvocadoTestSuite):
             # restart test again if it lost selenium grid location
             if ("urlopen error [Errno 113] No route to host" in cmd_output) or\
                     (("Terminated" in cmd_output) and ("Running 'kill" in cmd_output)):
-                self.machine_selenium.kill()
-                own_print("RESTARTING selenium grid and run test again")
+                own_print(f"RESTARTING selenium grid and run test again")
```

Comment:
Unnecessary f-string

Also, please prepend this commit with "test:", and squash it into the previous one, as it's a fixup of the "restart" commit.